### PR TITLE
update to goffice-0.10.43 - XPG6 bld - add test suite

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: components/library/goffice/Makefile [hunk 1]
```diff
@@ -36,28 +35,40 @@ include $(WS_MAKE_RULES)/ips.mk
...
 
-install:	$(INSTALL_32_and_64)
+install:    $(INSTALL_32_and_64)
 
 # To prevent "libtool_install_magic: unbound variable"
 unexport SHELLOPTS
 
-# TEST_32 fails test-math
-test:		$(TEST_32_and_64)
+test:       $(TEST_32_and_64)
```
Comment (AndWac) [inline components/library/goffice/Makefile:73]: Please mark the generated dependencies with a "# Auto-generated dependencies" comment like
gmake REQUIRED_PACKAGES automatically do. Hand-crafted dependencies should be separated (probably with its own comment).

Comment (ghost) [reply components/library/goffice/Makefile:73]: They were gmake REQUIRED_PACKAGES but with the 
"# Auto-generated dependencies" deleted.

Just add comment back?

**Comment by ghost** [comment]
Updated file from web - Not sure about squash commits from there

File: components/library/goffice/Makefile (hunk 1 above)
Comment (AndWac) [reply components/library/goffice/Makefile:73]: Yes, please.

**Pull request merged**